fe/fe-core/src/main/java/org/apache/doris/load/loadv2/LoadManager.java

```diff
@@ -109,7 +111,7 @@ public LoadManager(LoadJobScheduler loadJobScheduler) {
     public long createLoadJobFromStmt(LoadStmt stmt) throws DdlException {
         Database database = checkDb(stmt.getLabel().getDbName());
         long dbId = database.getId();
-        LoadJob loadJob = null;
+        LoadJob loadJob;
```

Comment:
Suggest use LoadJob loadjob = null,This approach is more in line with regulation.

Reply:
However, in Java8 and later version, there is no no need to do so, since the feature of Local Variable Type Inference.